Concrete::getById() must return only Concrete objects (#13548)
pimcore/pimcore: doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md

- [Config] The config node `pimcore.admin` and related parameters are moved to AdminBundle directly under `pimcore_admin` node. Please adapt your parameter usage accordingly eg. instead of `pimcore.admin.unauthenticated_routes`, it should be `pimcore_admin.unauthenticated_routes`
- [DataObjects] Removed deprecated preview url in class editor.
- [DataObjects] Removed sql filter functionality for data object grid

> - [DataObjects] Loading non-Concrete objects with the Concrete class is not any longer possible

Reviewer: ```suggestion
- [DataObjects] Loading non-Concrete objects with the Concrete class is no longer possible
```
